### ISSUE TITLE
Remove follower from their profile page

### ISSUE THREAD

**Issue by lapcat** [mastodon/mastodon#25039]
### Pitch

I'd like to see a menu option to remove a follower from their profile page, like there are menu options to mute, block, and report.

### Motivation

It's currently very difficult to remove a follower, without doing the old Twitter trick of block/unblock.

I know you can remove followers from the /relationships?relationship=followed_by page, but that's very inconvenient, because first you actually have to _find_ the follower there, and there's no search for followers. I personally have 28 pages of followers on that page, so you can see how that's a problem

**Comment by trwnh** [comment]
Duplicate of #8528
